molecularnodes/mda.py

```diff
@@ -428,18 +424,17 @@ def __init__(self, world_scale: float = 0.01, memory: bool = False):
         self.universe_reps = {}
         self.atom_reps = {}
         self.rep_names = []
-        self.uuid = str(uuid.uuid4().hex)
```

Comment:
This now breaks some of the tests that were checking the `uuid`, can you fix them?